components/forms/Label/Label.tsx

```diff
@@ -37,7 +37,7 @@ export const Label = (props: LabelProps): React.ReactElement => {
           ({t("required")})
         </span>
       )}
-      {required && <i className="visually-hidden">{t("required-field")}</i>}
+      {group && required && <i className="visually-hidden">{t("required-field")}</i>}
```

Comment:
What is this prop used for exactly ? If it's not used anywhere else should we remove it from the destructuring assignment from line 18.

Reply:
`group` is used to determine if the current element is a "single" element, like "input", or if it represents multiple elements, like checkboxes. It is already used on **Line 45**.